File: src/Frontend/heartmade-candles/src/types/Requests/UpdateCandleDetailsRequest.ts
import { CandleRequest } from "./CandleRequest";

export interface UpdateCandleDetailsRequest {
    candleRequest: CandleRequest;
    decorsIds: number[]; 
    layerColorsIds: number[];
    numberOfLayersIds: number[];
    smellsIds: number[];
    wicksIds: number[];
}

File: backend/HeartmadeCandles/HeartmadeCandles.API/Contracts/Requests/UpdateCandleDetailsRequest.cs
namespace HeartmadeCandles.API.Contracts.Requests
{
    public class UpdateCandleDetailsRequest
    {
        public CandleRequest CandleRequest { get; set; }
        public List<int> DecorsIds { get; set; }
        public List<int> LayerColorsIds { get; set; }
        public List<int> NumberOfLayersIds { get; set; }
        public List<int> SmellsIds { get; set; }
        public List<int> WicksIds { get; set; }
    }
}
